src/databricks/labs/ucx/source_code/base.py

```diff
@@ -183,11 +183,19 @@ def from_json(cls, json: dict) -> CurrentSessionState:
             catalog=json.get('catalog', DEFAULT_CATALOG),
             spark_conf=json.get('spark_conf', None),
             named_parameters=json.get('named_parameters', None),
-            data_security_mode=json.get('data_security_mode', None),
+            data_security_mode=CurrentSessionState._parse_security_mode(json.get('data_security_mode', None)),
```

Comment:
Now accessing the private method from outside the class, does the following work? What does mypy say

```suggestion
            data_security_mode=cls._parse_security_mode(json.get('data_security_mode', None)),
```